terraform/testnet/testnet/values.yaml

```diff
@@ -86,6 +86,8 @@ ingress:
   acm_certificate:
   wafAclArn:
   loadBalancerSourceRanges:
+  enableStickyness:
+  cookieDurationSeconds:
```

Comment:
can we specify a sane default? I believe it's 1 day (86400s) by default on ALB / TF

Reply:
sure